dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -132,6 +132,47 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
         m_notificationHandlers.remove(token);
     }
 
+    HRESULT __stdcall AppNotificationManager::Activate(
+        LPCWSTR /* appUserModelId */,
+        LPCWSTR invokedArgs,
+        [[maybe_unused]] NOTIFICATION_USER_INPUT_DATA const* data,
+        [[maybe_unused]] ULONG dataCount) noexcept try
+    {
+        winrt::IMap<winrt::hstring, winrt::hstring> userInput{ winrt::single_threaded_map<winrt::hstring, winrt::hstring>() };
+        for (unsigned long i = 0; i < dataCount; i++)
+        {
+            userInput.Insert(data[i].Key, data[i].Value);
+        }
+
+        winrt::AppNotificationActivatedEventArgs activatedEventArgs = winrt::make<winrt::Microsoft::Windows::AppNotifications::implementation::AppNotificationActivatedEventArgs>(invokedArgs, userInput);
+
+        // Need to store the first notification in the case of ToastActivation
+        if (!m_firstNotificationReceived)
+        {
+            m_firstNotificationReceived = true;
+
+            std::wstring commandLine{ GetCommandLine() };
+            auto pos{ commandLine.find(c_notificationActivatedArgument) };
+            if (pos == std::wstring::npos)
```

Comment:
Add a comment here to give context. 
// If the app was not launched due to a toast invoked event, we will invoke the event handler. Otherwise, we populate the EventArgs and signal the Main thread that we are ready to access the EventArgs.